node_modules/oae-core/userdetails/js/userdetails.js

```diff
@@ -0,0 +1,161 @@
+/*!
+ * Copyright 2015 Apereo Foundation (AF) Licensed under the
+ * Educational Community License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License. You may
+ * obtain a copy of the License at
+ *
+ *     http://opensource.org/licenses/ECL-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an "AS IS"
+ * BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+
+define(['jquery', 'oae.core'], function($, oae) {
+
+    return function(uid) {
+
+        // The widget container
+        var $rootel = $('#' + uid);
+
+        /**
+         * Edit the profile
+         */
+        var editUserDetails = function() {
+            // Disable the form
+            $('#userdetails-form *', $rootel).prop('disabled', true);
+
+            // Accept the terms and conditions
+            oae.api.user.acceptTC(function(err) {
```

Comment:
It looks like this causes an error if T&C is not enabled, and the user cannot continue using the system.